crates/chainspec/src/spec.rs

```diff
@@ -758,6 +758,8 @@ impl From<Genesis> for ChainSpec {
             (OptimismHardfork::Ecotone.boxed(), genesis_info.ecotone_time),
             #[cfg(feature = "optimism")]
             (OptimismHardfork::Fjord.boxed(), genesis_info.fjord_time),
+            #[cfg(feature = "optimism")]
+            (OptimismHardfork::Granite.boxed(), genesis_info.granite_time),
```

Comment:
this only adds it to base-sepolia?

we should also add it to op-sepolia?

Reply:
This change should apply Granite to any ChainSpec based on the genesis file config. The base-sepolia and op-sepolia activation timestamps are configured towards the end of `crates/ethereum-forks/src/hardfork/optimism.rs`